sycl/doc/CompilerAndRuntimeDesign.md

```diff
@@ -918,8 +918,8 @@ space attributes in SYCL mode:
 | Address space attribute | SYCL address_space enumeration |
 |-------------------------|--------------------------------|
 | `__attribute__((opencl_global))` | global_space, constant_space |
-| `__attribute__((opencl_global_host))` | global_host_space |
-| `__attribute__((opencl_global_device))` | global_device_space |
+| `__attribute__((opencl_global_host))` | ext_intel_global_host_space |
+| `__attribute__((opencl_global_device))` | ext_intel_global_device_space |
```

Comment:
Where are these values defined?

Reply:
Hm, they are not defined.

Reply:
They are defined here: https://github.com/intel/llvm/pull/4432/files#diff-50b7a2fc16e4a14f34dd0bbd5d6a46ee8afe93735d289120b07d0e800da981faR49

Reply:
> They are defined here: https://github.com/intel/llvm/pull/4432/files#diff-50b7a2fc16e4a14f34dd0bbd5d6a46ee8afe93735d289120b07d0e800da981faR49

What I mean is: where can users find the description of these SYCL address spaces? I expect there should be an extension defining them as they are not part of the core specification.

Reply:
Updated SYCL_EXT_INTEL_USM_ADDRESS_SPACES spec in https://github.com/intel/llvm/pull/4432/commits/8315b0138ef73f2a7e19a320c57ea8d4e32b695f

`device_space` -> `ext_intel_global_device_space`
`host_space` -> `ext_intel_global_host_space`